.github/workflows/alpha-release.yml

```diff
@@ -20,17 +20,17 @@ jobs:
           node-version: 16.x
           cache: 'yarn'
       - name: Install dependencies for master
```

Comment:
you may be interested in this way of abstracting the boilerplate from pnpm + cache, etc:
https://github.com/NullVoxPopuli/baobab-ram/pull/1/files#diff-423ef59ce8dec1c2647bc93466fa66fd1eb65a9a65a631d4730fdb753b0ef792